examples/python/wait_for_ready/wait_for_ready_with_client_timeout_example_server.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python serer of utilizing wait-for-ready flag with client time out."""
+
+from concurrent import futures
+import logging
+from time import sleep
+
+import grpc
+
+helloworld_pb2, helloworld_pb2_grpc = grpc.protos_and_services(
+    "helloworld.proto")
+
+_INITIAL_METADATA = ((b'initial-md', 'initial-md-value'),)
+
+
+class Greeter(helloworld_pb2_grpc.GreeterServicer):
+
+    def SayHelloStreamReply(self, request, servicer_context):
+        # Send initial metadata back to indicate server is ready and running.
+        print("sleeping 5s before sending metadata back")
+        sleep(5)
+
+        # Initial metadata will be send back immediately after calling send_initial_metadata.
+        print("sending inital metadata back")
+        servicer_context.send_initial_metadata(_INITIAL_METADATA)
+
+        # Server can do whatever it wants here before send actual response.
+        yield helloworld_pb2.HelloReply(message='Hello, %s!' % (request.name))
```

Comment:
If this example is going to be a streaming RPC, we should probably be returning multiple messages. Honestly, I think this example would be fine as a unary RPC.

Reply:
It's easier on client side to process stream response, I'll change here to return multiple messages.